html/includes/modal/new_alert_rule.inc.php

```diff
@@ -136,7 +136,7 @@
         <div class='form-group'>
             <label for='proc' class='col-sm-3 control-label'>Procedure URL: </label>
             <div class='col-sm-9'>
-                <input type='text' id='proc' name='proc' class='form-control' maxlength='30'>
+                <input type='text' id='proc' name='proc' class='form-control' maxlength='1024'>
```

Comment:
The database only allows for 80 characters.
So set that here, or update the database ;)